Classes/Private/Manager/Audio/OCTAudioEngine+Private.h

```diff
@@ -11,68 +11,21 @@
 
-/**
- * OCTAudioEngine functions
- */
-extern OSStatus (*_NewAUGraph)(AUGraph *outGraph);
-extern OSStatus (*_AUGraphAddNode)(
-    AUGraph inGraph,
-    const AudioComponentDescription *inDescription,
-    AUNode *outNode);
-extern OSStatus (*_AUGraphOpen)(AUGraph inGraph);
-extern OSStatus (*_AUGraphNodeInfo)(AUGraph inGraph,
-                                    AUNode inNode,
-                                    AudioComponentDescription *outDescription,
-                                    AudioUnit *outAudioUnit);
-extern OSStatus (*_AUGraphStart)(AUGraph inGraph);
-extern OSStatus (*_AUGraphStop)(AUGraph inGraph);
-extern OSStatus (*_AUGraphInitialize)(AUGraph inGraph);
-extern OSStatus (*_AUGraphUninitialize)(AUGraph inGraph);
-extern OSStatus (*_AUGraphIsRunning)(AUGraph inGraph, Boolean *outIsRunning);
-
-extern OSStatus (*_AudioUnitSetProperty)(AudioUnit inUnit,
-                                         AudioUnitPropertyID inID,
-                                         AudioUnitScope inScope,
-                                         AudioUnitElement inElement,
-                                         const void *inData,
-                                         UInt32 inDataSize
-);
-extern OSStatus (*_AudioUnitRender)(AudioUnit inUnit,
-                                    AudioUnitRenderActionFlags *ioActionFlags,
-                                    const AudioTimeStamp *inTimeStamp,
-                                    UInt32 inOutputBusNumber,
-                                    UInt32 inNumberFrames,
-                                    AudioBufferList *ioData
-);
-
-extern OSStatus (*_DisposeAUGraph)(AUGraph inGraph);
-
-@interface OCTAudioEngine (Private)
-
-@property (nonatomic, assign) AUGraph processingGraph;
-@property (nonatomic, assign) AudioUnit ioUnit;
-@property (nonatomic, assign) TPCircularBuffer outputBuffer;
-@property (nonatomic, assign) TPCircularBuffer inputBuffer;
-@property (nonatomic, assign) OCTToxAVSampleRate inputSampleRate;
-@property (nonatomic, assign) OCTToxAVSampleRate outputSampleRate;
-
-- (BOOL)fillError:(NSError **)error
-         withCode:(NSUInteger)code
-      description:(NSString *)description
-    failureReason:(NSString *)failureReason;
-
-OSStatus inputRenderCallBack(void *inRefCon,
-                             AudioUnitRenderActionFlags  *ioActionFlags,
-                             const AudioTimeStamp    *inTimeStamp,
-                             UInt32 inBusNumber,
-                             UInt32 inNumberFrames,
-                             AudioBufferList *ioData);
-
-OSStatus outputRenderCallBack(void *inRefCon,
-                              AudioUnitRenderActionFlags *ioActionFlags,
-                              const AudioTimeStamp *inTimeStamp,
-                              UInt32 inBusNumber,
-                              UInt32 inNumberFrames,
-                              AudioBufferList *ioData);
+extern int kBufferLength;
+extern int kNumberOfInputChannels;
```

Comment:
This number is used for output as well. `kNumberOfInputChannels` -> `kNumberOfChannels`.